StreamChatUITestsAppUITests/Tests/Message Delivery Status/MessageDeliveryStatus+ChannelList_Tests.swift

```diff
@@ -0,0 +1,305 @@
+        }
+        WHEN("participant reads the user's message") {
+            participantRobot.readMessage()
+        }
+        THEN("user spots double checkmark next to the message") {
+            userRobot.assertMessageDeliveryStatusInChannelPreview(.read)
+        }
+        AND("read count is hidden") {
+            userRobot.assertMessageReadCountInChannelPreview(readBy: 0)
+        }
+    }
+
+    func test_deliveryStatusHiddenInPreview_whenMessageIsSentAndReadEventsIsDisabled() {
+        linkToScenario(withId: 170)
+
+        GIVEN("user opens chat") {
+            backendRobot.setReadEvents(to: false)
+            userRobot.login()
+            userRobot.openChannel()
+        }
+        AND("user sends a new message") {
+            userRobot.sendMessage(message)
+        }
+        AND("message is succesfully sent") {
+            userRobot.waitForNewMessage(withText: message)
+        }
+        WHEN("user retuns to the channel list") {
+            userRobot.tapOnBackButton()
+        }
+        THEN("delivery status is hidden") {
+            userRobot
+                .assertMessageDeliveryStatus(.failed)
+                .assertMessageReadCount(readBy: 0)
+        }
+    }
+
+    func test_deliveryStatusHiddenInPreview_whenMessageIsSentByParticipant() {
+        linkToScenario(withId: 171)
+
+        GIVEN("user opens chat") {
+            userRobot
+                .login()
+                .openChannel()
+        }
+        WHEN("participant sends a new message") {
+            participantRobot
+                .sendMessage(message)
+                .waitForNewMessage(withText: message)
+        }
+        AND("user retuns to the channel list") {
+            userRobot.tapOnBackButton()
+        }
+        THEN("delivery status is hidden") {
+            userRobot
+                .assertMessageDeliveryStatus(.failed)
+                .assertMessageReadCount(readBy: 0)
+        }
+    }
+}
+
+// MARK: Thread Reply
+
+extension MessageDeliveryStatus_ChannelList_Tests {
+
+    func test_singleCheckmarkShownInPreview_whenThreadReplyIsSent() {
+        linkToScenario(withId: 172)
+
+        GIVEN("user opens chat") {
+            userRobot
+                .login()
+                .openChannel()
+        }
+        AND("participant sends a new message") {
+            participantRobot.sendMessage(message)
+        }
+        AND("user replies to the message in thread") {
+            userRobot.replyToMessageInThread(threadReply)
+        }
+        WHEN("user retuns to the channel list") {
+            userRobot
+                .tapOnBackButton()
+                .tapOnBackButton()
+        }
+        THEN("delivery status is hidden") {
+            userRobot
+                .assertMessageDeliveryStatusInChannelPreview(.sent)
+                .assertMessageReadCountInChannelPreview(readBy: 0)
+        }
+    }
+
+    func test_errorIndicatorShownInPreview_whenThreadReplyFailedToBeSent() {
```

Comment:
Hm, we do not show thread replies in channel preview? 🤔 Or is it the one with `Send also to main channel` checkmark selected?

Reply:
@evsaev we do show errors for thread replies ->

https://user-images.githubusercontent.com/4924709/167661886-0aaa36bd-55f8-4d6b-a260-037c08b61482.mp4

Reply:
@bielikb it's a bug. We should never show thread reply in preview even in case of sending failure. That's aligned with unread counts when thread replies do not increment it. Basically, they should not affect channel preview (the typing indicator should also be fixed). That's the current approach which is a bit confusing, but that's what Android and other platforms do.

Reply:
Investigating why it's happening